doc/changes/latest.inc

```diff
@@ -47,6 +47,8 @@ Enhancements
 
 - Add projections when printing a :class:`mne.Info` in the notebook (:gh:`9403` by `Alex Gramfort`_)
 
+- Add support for SURE parameter selection in :func:`mne.inverse_sparse.mixed_norm` and make ``alpha`` parameter now default to ``'sure'`` (:gh:`9430` by **new contributor** `Pierre-Antoine Bannier`_ and `Alex Gramfort`_)
```

Comment:
New contributor line needs to be updated with RST substitution, see/follow examples above (eg., David Julien)

Please also move Richards' get_chpi_info line below the new contributor ones

Reply:
(also please move this line to the top with the other new contributor lines)